src/composables/PluginManager.js

```diff
@@ -215,8 +215,15 @@ export async function addNewComponent(
   plugin,
   path,
   definition,
+  position = null,
```

Comment:
Also, why not also use new param `position` when calling `addNewTemplateComponent` ?

Reply:
Done

Reply:
> Also, why not also use new param `position` when calling `addNewTemplateComponent` ?
Since the position is defined by the drawer, I want to use the `position` parameter when I need to force it.